src/compiler/rust/canister_methods/src/lib.rs

```diff
@@ -205,35 +211,23 @@ pub fn canister_methods(_: TokenStream) -> TokenStream {
             }
         });
 
-    let reload_js = reload_js::get_reload_js(&compiler_info.env_vars);
+    let reload_js = reload_js::get_reload_js();
 
     let upload_file_chunk = upload_file_chunk::get_upload_file_chunk();
 
     quote! {
-        static ASSETS_DIR: Dir = include_dir!("$CARGO_MANIFEST_DIR/src/assets");
-
         #init_method
 
         #post_upgrade_method
 
-        #pre_upgrade_method
-
-        #inspect_message_method
-
-        #heartbeat_method
-
-        #(#query_methods)*
-
-        #(#update_methods)*
-
         #[ic_cdk_macros::query]
         fn __get_candid_interface_tmp_hack() -> String {
             std::str::from_utf8(CANDID)
                 .expect("candid.did could not be read")
                 .to_string()
         }
 
-        fn initialize_js(js: &str, init: bool) {
+        fn initialize_js(js: &str, init: bool, function_index: i32, pass_arg_data: i32) {
```

Comment:
Resolved?

Reply:
yes, it's just an i32 version of the boolean value that was already there, if there is arg data to pass this will be 1, then if the value is 0 we will not waste time getting arg data and can just move onto the meat of the method